releasenotes/notes/rpc_package_index-a03d4d131c16d710.yaml

```diff
@@ -0,0 +1,12 @@
+---
+upgrade:
+  - The ``repo_build_pip_extra_index`` should be
+    replaced by a new variable (list type) named
+    ``repo_build_pip_extra_indexes``. This is required
+    if the deployer uses a python package index in addition
+    to the standard PyPI.
+
+deprecations:
+  - The ``repo_build_pip_extra_index`` variable is now
+    deprecated, and the deployer should use the list
+    ``repo_build_pip_extra_indexes`` instead.
```

Comment:
ya, I'd say the upgrade/deprecation is not needed here and is covered by OSA.  A simple note stating we are using the variable should be sufficient.